src/page-modules/contact/components/form/input.tsx

```diff
@@ -22,12 +26,12 @@ export const Input = ({
   name,
   checked,
   value,
-  description,
+  modalContent,
   onChange,
 }: InputProps) => {
   const { t } = useTranslation();
   const [isModalOpen, setIsModalOpen] = useState(false);
-  const displayDescriptionModal = description && isModalOpen; // Condition to avoid rendering of useEffect inside DescriptionModal before opened.
+  const displayDescriptionModal = modalContent && isModalOpen; // Condition to avoid rendering of useEffect inside DescriptionModal before opened.
```

Comment:
Should you check if the modalContent actually has one of the properties set? None of the modalContent properties are currently required.

Reply:
Right. Yes, at least on of the properties inside modal content should be present. Will update.